docs/samples/git-setup/policytemplates/version_4.Y.Z/sno-ran-full-du-profile/sno-ran-full-du-pg-v4-Y-Z-v1.yaml

```diff
@@ -0,0 +1,349 @@
+    - path: source-crs/PtpSubscriptionOperGroup.yaml
+    - path: source-crs/PtpOperatorStatus.yaml
+    # SRIOV operator
+    - path: source-crs/SriovSubscriptionNS.yaml
+    - path: source-crs/SriovSubscriptionOperGroup.yaml
+    - path: source-crs/SriovSubscription.yaml
+      patches:
+      - spec:
+          source: redhat-operators
+          installPlanApproval:
+            '{{hub $configMap:=(lookup "v1" "ConfigMap" "" (printf "%s-pg" .ManagedClusterName)) hub}}{{hub dig "data" "install-plan-approval" "Manual" $configMap hub}}'
+    - path: source-crs/SriovOperatorStatus.yaml
+    # SRIOV Accelerator
+    - path: source-crs/AcceleratorsNS.yaml
+    - path: source-crs/AcceleratorsOperGroup.yaml
+    - path: source-crs/AcceleratorsSubscription.yaml
+      patches:
+      - spec:
+          channel: "stable"
+          source: certified-operators
+          installPlanApproval:
+            '{{hub $configMap:=(lookup "v1" "ConfigMap" "" (printf "%s-pg" .ManagedClusterName)) hub}}{{hub dig "data" "install-plan-approval" "Manual" $configMap hub}}'
+    - path: source-crs/AcceleratorsOperatorStatus.yaml
+    # LCA
+    - path: source-crs/LcaSubscriptionNS.yaml
+    - path: source-crs/LcaSubscriptionOperGroup.yaml
+    - path: source-crs/LcaSubscription.yaml
+      patches:
+      - spec:
+          source: redhat-operators
+          installPlanApproval:
+            '{{hub $configMap:=(lookup "v1" "ConfigMap" "" (printf "%s-pg" .ManagedClusterName)) hub}}{{hub dig "data" "install-plan-approval" "Manual" $configMap hub}}'
+    - path: source-crs/LcaSubscriptionOperGroup.yaml
+    # OADP
+    - path: source-crs/OadpSubscriptionNS.yaml
+    - path: source-crs/OadpSubscriptionOperGroup.yaml
+    - path: source-crs/OadpSubscription.yaml
+      patches:
+      - spec:
+          source: redhat-operators
+          installPlanApproval:
+            '{{hub $configMap:=(lookup "v1" "ConfigMap" "" (printf "%s-pg" .ManagedClusterName)) hub}}{{hub dig "data" "install-plan-approval" "Manual" $configMap hub}}'
+    - path: source-crs/OadpOperatorStatus.yaml
+    # Local storage operator
+    - path: source-crs/StorageNS.yaml
+    - path: source-crs/StorageOperGroup.yaml
+    - path: source-crs/StorageSubscription.yaml
+      patches:
+      - spec:
+          source: redhat-operators
+          installPlanApproval:
+            '{{hub $configMap:=(lookup "v1" "ConfigMap" "" (printf "%s-pg" .ManagedClusterName)) hub}}{{hub dig "data" "install-plan-approval" "Manual" $configMap hub}}'
+    - path: source-crs/StorageOperatorStatus.yaml
+# CONFIGURATION POLICY
+- name: v1-config-policy
+  manifests:
+    - path: source-crs/DisableOLMPprof.yaml
+    - path: source-crs/DisableSnoNetworkDiag.yaml
+    # REDUCE FOOTPRINT
+    - path: source-crs/ReduceMonitoringFootprint.yaml
+      patches:
+      - data:
+          config.yaml: |
+            alertmanagerMain:
+              enabled: false
+            nodeExporter:
```

Comment:
where did this come from. We do not want all of these enabled on the managed cluster

Reply:
These come from values set by ACM observability. We disable ACM's ability to write this configmap because of conflicting settings for this "yaml in a string" value, but we merged our content with what ACM sets. The collectors entries below each have a boolean enable/disable value. With this setting only netclass and netdev are enabled which matches the state when the configmap is removed entirely. 

For reference, the node-exporter pod command line looks like this with the nodeExporter config in this PR.
```
    - --no-collector.wifi
    - --collector.filesystem.mount-points-exclude=^/(dev|proc|sys|run/k3s/containerd/.+|var/lib/docker/.+|var/lib/kubelet/pods/.+)($|/)
    - --collector.netclass.ignored-devices=^(veth.*|[a-f0-9]{15}|enP.*|ovn-k8s-mp[0-9]*|br-ex|br-int|br-ext|br[0-9]*|tun[0-9]*|cali[a-f0-9]*)$
    - --collector.netdev.device-exclude=^(veth.*|[a-f0-9]{15}|enP.*|ovn-k8s-mp[0-9]*|br-ex|br-int|br-ext|br[0-9]*|tun[0-9]*|cali[a-f0-9]*)$
    - --collector.cpu.info
    - --collector.textfile.directory=/var/node_exporter/textfile
    - --no-collector.btrfs
    - --runtime.gomaxprocs=0
    - --no-collector.cpufreq
    - --no-collector.tcpstat
    - --collector.netdev
    - --collector.netclass
    - --collector.netclass.netlink
    - --no-collector.buddyinfo
    - --no-collector.mountstats
    - --no-collector.ksmd
    - --no-collector.processes
    - --no-collector.systemd
```